### PR TITLE
Make the alignment of composer modal errors to center

### LINKED ISSUE #6199: Alignment of composer modal errors
### Steps to reproduce

1. Send a message to an invalid recipient

### Expected behavior

Well designed error screen.

### Actual behavior

Bildschirmfoto vom 2022-04-07 20-41-29

### Mail app version

1.x

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
fixes #6199

### CODE REVIEW

File: src/components/Composer.vue [hunk 1]
```diff
@@ -306,7 +306,7 @@
 	<EmptyContent v-else-if="state === STATES.DISCARDED" icon="icon-mail">
 		<h2>{{ t('mail', 'Draft was discarded!') }}</h2>
 	</EmptyContent>
-	<div v-else-if="state === STATES.ERROR" class="emptycontent" role="alert">
```
Comment (ChristophWurst) [inline src/components/Composer.vue:309]: line 1176 targets `.emptycontent`. I gues that rule can be dropped now

**Comment by ChristophWurst** [comment]
@JuliaKirschenheuter :ping_pong: 

Please drop the dead code and rebase the branch to resolve the conflicts

**Comment by JuliaKirschenheuter** [comment]
> Please drop the dead code and rebase the branch to resolve the conflicts

done

**Review by GretaD — APPROVED** [review]
needs rebase to resolve conflicts, but approving so its not blocked

**Comment by ChristophWurst** [comment]
@JuliaKirschenheuter please rebase another time :see_no_evil:

**Pull request merged**